SunRise/Controllers/Library/LibraryCell/Favorite/FavoriteCell/FavoriteTableViewCell.swift

```diff
@@ -0,0 +1,54 @@
+import UIKit
+
+final class FavoriteTableViewCell: UITableViewCell, Reusable, NibLoadable {
+
+    @IBOutlet weak private var songImage: UIImageView!
+    @IBOutlet weak private var titleLabel: UILabel!
+    @IBOutlet weak private var userLabel: UILabel!
+    @IBOutlet weak private var favoriteButton: UIButton!
+    
+    var isFavoriteButtonPressed: (() -> Void)?
+    var isDetailButtonPressed: (() -> Void)?
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+    }
+    
+    override func prepareForReuse() {
+        songImage.image = .none
```

Comment:
thêm super

Reply:
Đoạn này thông báo lỗi anh "Value of type 'UITableViewCell' has no member 'songImage'"